src/utils/axiosInterceptors.ts

```diff
@@ -206,7 +206,8 @@ const retryRequests = (err: any): any => {
   }
   if (shouldRetry(err.response.status)) {
     err.config.retriesCount = err.config.retriesCount | 0;
-    const shouldRetry = err.config.retriesCount < RETRIES;
+    const minRetries = err.config.retries || RETRIES;
```

Comment:
This var should be probably named `maxRetries`

Reply:
True, I already changed it. Thanks